libcloud/compute/drivers/outscale.py

```diff
@@ -270,7 +312,7 @@ def create_node(self,
                     ex_keypair_name: str = None,
                     ex_max_vms_count: int = None,
                     ex_min_vms_count: int = None,
-                    ex_nics: dict = None,
+                    ex_nics: [dict] = None,
```

Comment:
Hm, does ``[foo]`` type annotation work correctly? I never really seen it used before.

I mostly use ``List[type]`` or ``list[type]``).

Reply:
EDIT: I just confirmed, it indeed doesn't appear to work:

```bash
libcloud/compute/base.py:216: error: Bracketed expression "[...]" is not valid as a type
```

Please update all ``[foo]`` type annotations to use ``List[foo]`` instead (``List`` is available in ``typing`` module).